theme.json

```diff
@@ -1051,6 +1051,67 @@
 					"fontSize": "var:preset|font-size|x-large"
 				}
 			},
+			"core/quote": {
+				"border": {
+					"left": {
+						"color": "var:preset|color|contrast",
+						"style": "solid",
+						"width": "2px"
+					},
+					"right": {
+						"color": "var:preset|color|contrast",
+						"style": "solid",
+						"width": "2px"
+					}
+				},
+				"spacing": {
+					"blockGap": "var:preset|spacing|30",
+					"margin": {
+						"left": "0",
+						"right": "0"
+					},
+					"padding": {
+						"top": "var:preset|spacing|30",
+						"right": "var:preset|spacing|40",
+						"bottom": "var:preset|spacing|30",
+						"left": "var:preset|spacing|40"
+					}
+				},
+				"typography": {
+					"fontSize": "var:preset|font-size|medium",
+					"lineHeight": "1.1",
+					"fontWeight": "600"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "0.65em",
+							"fontStyle": "normal",
+							"fontWeight": "400"
+						},
+						"css": "& sub { font-size: 0.65em }"
```

Comment:
I would have preferred to use preset values here, but these seem to need to scale with the base font size of the block based on the designs.